src/DynamoInstall/Product.wxs

```diff
@@ -79,7 +79,7 @@
     <DirectoryRef Id="TARGETDIR">
       <Component Id="ProductRegistry" Win64="yes">
         <RegistryKey Root="HKLM"
-                     Key="SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(var.ProductName) $(var.Major).$(var.Minor)"
+                     Key="SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(var.ProductName) $(var.Major)"
```

Comment:
This make sure when uninstalling, uninstaller looks for major version folder only as well, the uninstall logic is not touched, so side by side install/uninstall is still valid